account_invoice_margin/models/account_move_line.py

```diff
@@ -4,55 +4,6 @@
 from odoo import api, fields, models
 
 
-class AccountMove(models.Model):
```

Comment:
This change is not correct, as it was that way on purpose, as models coming from one2many are allowed and even recommended to be kept on the same file.

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#file-naming

Reply:
Oh, I always forget about that. My personal taste differs here.
Anyway, reverted.